hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -3923,6 +3923,19 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.client.ec.write.failed.blocks.tolerated</name>
+  <value>2147483647</value>
```

Comment:
What does this big number mean? Can the default value be -1？

Reply:
Hi,  @zhangshuyan0 . Thanks very much for reviewing.  Yes, the default value can be -1. I have modify the default value and description in hdfs-default.xml. Thanks.